program/src/state.rs

```diff
@@ -76,6 +222,11 @@ pub(crate) fn collect_holder_rewards_pool_signer_seeds<'a>(
 #[derive(Clone, Copy, Debug, Default, PartialEq, Pod, ShankAccount, Zeroable)]
 #[repr(C)]
 pub struct HolderRewards {
+    /// The rewards per token exchange rate when this holder last harvested.
+    ///
+    /// Stored as a `u128`, which includes a scaling factor of `1e9` to
+    /// represent the exchange rate with 9 decimal places of precision.
+    pub last_rewards_per_token: u128,
```

Comment:
nit: I wonder if it might be clearer to call this `last_accumulated_rewards_per_token` -- the idea that it's the *accumulated* value per token might make things clearer